pavement.py

```diff
@@ -119,5 +119,5 @@ def release():
     sh("git tag -s '%(name)s-%(version)s' -m 'Version bump to %(version)s'" % {'name': NAME, 'version': VERSION})
     sh("git push --tags")
     sh("git push")
-    sh("python setup.py register sdist upload")
```

Comment:
I’m not completely sure whether we should remove `register`. Is it deprecated as well?

Reply:
I'm not sure but works without that. But we have already register. I will get this back. 

https://docs.python.org/3.1/distutils/packageindex.html

Reply:
Fixed, I added `register` back.

Reply:
It's true the `register` is only needed once...